docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./data:/var/lib/postgresql/data/
+    environment:
+      - "POSTGRES_HOST_AUTH_METHOD=trust"
+
+  backend_shoes_store:
+    build:
+      context: ./backend_shoes_store
+    container_name: backend
+    command: /code/start-dev.sh
+    volumes:
+      - ./backend_shoes_store:/code
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+
+  frontend_shoes_store:
```

Comment:
Quando tentamos rodar o front não funcionou. Poderia verificar?

Reply:
Subi um Fix, poderia tentar novamente?

Fix baseado [nesta](https://stackoverflow.com/questions/55541041/node-modules-missing-even-after-npm-install-in-docker-container) discussao do stack overflow.